### ISSUE TITLE
Completion should be smarter if you type a font name with a space

### ISSUE THREAD

**Issue by njx** [adobe/brackets-edge-web-fonts#22]
1. Bring up the EWF code hint menu
2. Type "andale" and then hit space

Actual result: EWF assumes you're starting a new font name.
Expected result: It would be nice if it were smart and assumed you were still typing the same font (since you haven't typed a comma), and add the quotes for you if you then accept a completion from the menu.

**Comment by joelrbrandt** [comment]
I agree. It's worth noting, however, that it does work properly if you start with a quote character. That is, if you've started with a quote, typing a space keeps you in the same token.

It also works right if you DON'T start with a quote, but then complete to a name with a space (before actually typing a space character). So, I think this is pretty low priority. Marking as such.

**Comment by iwehrman** [comment]
I'll look into this.

**Comment by couzteau** [comment]
PR was not sufficient covering all edge cases

**Comment by couzteau** [comment]
This should be closed. Fix has been merged

**Comment by couzteau** [comment]
Fix merged https://github.com/adobe/brackets-edge-web-fonts/pull/149
